cluster/gce/gci/mounter/stage-upload.sh

```diff
@@ -39,29 +39,29 @@ ACI_DIR=${STAGING_DIR}/gci-mounter
 CWD=${PWD}
 
 # Cleanup the temporary directories
-function cleanup {
-    rm -rf ${DOWNLOAD_DIR}
-    rm -rf ${STAGING_DIR}
-    cd ${CWD}
+cleanup() {
+    rm -rf "${DOWNLOAD_DIR}"
+    rm -rf "${STAGING_DIR}"
+    cd "${CWD}"
 }
 
 # Delete temporary directories on exit
 trap cleanup EXIT
 
-mkdir ${ACI_DIR}
+mkdir "${ACI_DIR}"
 
 # Convert docker image to aci and stage it
 echo "Downloading docker2aci ${DOCKER2ACI_VERSION}"
-wget "https://github.com/appc/docker2aci/releases/download/${DOCKER2ACI_VERSION}/docker2aci-${DOCKER2ACI_VERSION}.tar.gz" &> /dev/null
+wget "https://github.com/appc/docker2aci/releases/download/${DOCKER2ACI_VERSION}/docker2aci-${DOCKER2ACI_VERSION}.tar.gz" "&>" /dev/null
```

Comment:
/assign @BenTheElder 
Is this correct? I'm not sure.

Reply:
```
In ./cluster/gce/gci/mounter/stage-upload.sh line 55:
wget "https://github.com/appc/docker2aci/releases/download/${DOCKER2ACI_VERSION}/docker2aci-${DOCKER2ACI_VERSION}.tar.gz" &> /dev/null
                                                          ^----------^ SC2039: In POSIX sh, &> is undefined.
Ref: https://github.com/koalaman/shellcheck/wiki/SC2039
```

Reply:
This is not correct, `&> /dev/null` is shell redirection in bash to redirect both stdout and stderr to `/dev/null`. In POSIX sh `>/dev/null 2>&1` is the correct equivilant.

Reply:
YES, but why 2>&1?

Reply:
has been updated.

Reply:
the file descriptors for stdin, stdout, and stderr have well known numbers: https://en.wikipedia.org/wiki/File_descriptor

`2>&1` redirects stderr to stdout, `>/dev/null 2>&1` redirects stdout to `/dev/null` and then stderr to stdout.

Reply:
has been updated.